DriverAchievementUI/src/main/java/com/drivequant/drivekit/driverachievement/ui/streaks/viewmodel/StreaksListViewModel.kt

```diff
@@ -2,28 +2,26 @@ package com.drivequant.drivekit.driverachievement.ui.streaks.viewmodel
 
 import android.arch.lifecycle.MutableLiveData
 import android.arch.lifecycle.ViewModel
-import com.drivequant.drivekit.common.ui.DriveKitUI
 import com.drivequant.drivekit.core.DriveKit
 import com.drivequant.drivekit.databaseutils.entity.Streak
-import com.drivequant.drivekit.databaseutils.entity.StreakTheme
 import com.drivequant.drivekit.driverachievement.DriveKitDriverAchievement
-import com.drivequant.drivekit.driverachievement.streak.AchievementSyncStatus
-import com.drivequant.drivekit.driverachievement.streak.StreaksQueryListener
+import com.drivequant.drivekit.driverachievement.StreakSyncStatus
+import com.drivequant.drivekit.driverachievement.StreaksQueryListener
 import com.drivequant.drivekit.driverachievement.ui.DriverAchievementUI
 
 class StreaksListViewModel : ViewModel() {
 
     var filteredStreaksData: MutableList<StreaksData> = mutableListOf()
     val streaksData: MutableLiveData<List<StreaksData>> = MutableLiveData()
-    var syncStatus: AchievementSyncStatus = AchievementSyncStatus.NO_ERROR
+    var syncStatus: StreakSyncStatus = StreakSyncStatus.NO_ERROR
 
     fun fetchStreaks() {
         if (DriveKit.isConfigured()) {
             DriveKitDriverAchievement.getStreaks(object : StreaksQueryListener {
                 override fun onResponse(
-                    achievementSyncStatus: AchievementSyncStatus,
+                    streakSyncStatus: StreakSyncStatus,
                     streaks: List<Streak>) {
-                    syncStatus = achievementSyncStatus
+                    syncStatus = streakSyncStatus
                     filteredStreaksData = getFilteredStreaks(streaks)
                     streaksData.postValue(filteredStreaksData)
                 }
```

Comment:
@mohamed-garci We need to check with iOS -> all images must be prefixed with "dk_"

Reply:
@Steven-DriveQuant We kept the same keys because we use the same badge endpoint.

Reply:
Les noms sont envoyés par le backend, ils ne peuvent pas être modifiés côté backend pour être rétrocompatibles